### PR TITLE
Feature/add frontend

### PR DESCRIPTION
**Pull request by Eernie**

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Frontend
+
+This project was generated with [angular-cli](https://github.com/angular/angular-cli) version 1.0.0-beta.11-webpack.2.
```
Comment (Jurrie) [inline frontend/README.md]: Isn't there something better to write in this file than the default generated content?

File: frontend/src/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Frontend</title>
+  <base href="/">
```
Comment (Jurrie) [inline frontend/src/index.html]: This is the default base, isn't it? Why include the <base /> tag then?

Comment (Eernie) [inline frontend/src/index.html]: Angular 2 needs this take for the [router.](https://angular.io/docs/ts/latest/guide/router.html#!#base-href) If we remove this, we can't make use of the `pushState` which we will need for routing to the right dasboards and such.

File: frontend/README.md (hunk 1 above)
Comment (Eernie) [inline frontend/README.md]: @Jurrie: In the future we need a `readme.md` in the root of this repo to explain the project. For now we can do with this generated version I guess, unless you're missing something

**Pull request merged**